packages/application/style/core.css

```diff
@@ -0,0 +1,87 @@
+/*-----------------------------------------------------------------------------
```

Comment:
here's the new file to implement the change discussed here: https://github.com/jupyterlab/jupyterlab/pull/11944#issuecomment-1024007394